eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -33,79 +33,86 @@ parameters:
     type: string
     default: ''
 steps:
-- template: /eng/common/pipelines/templates/steps/enable-long-path-support.yml
+  - ${{ if eq(length(parameters.PackageInfoLocations), 0) }}:
```

Comment:
Same ask. Why batch work has to make this change?

Also, why don't we use one line conditional check?
```
- ${{ if ne(length(parameters.PackageInfoLocations), 0) }}:
```

Reply:
Spring.   They add `skipPublishDocMs: true` to all their artifacts:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/ci.yml

It's causing a failure in their nightlies:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1540590&view=logs&j=f8e040b3-c0ff-5789-0eda-99daaaa3fc1b&t=0c1ba28b-ea79-5c16-1c0c-62e27ae6ce8d

Reply:
> Also, why don't we use one line conditional check?

A one line conditional would make for an odd log.   It would look like the job started, then did nothing.  I'd prefer we write something in the logs, even if it's just "Skipping DocsMS Update because package list was empty".

Reply:
We make this check here because it's the first place where we've reduced the yaml parameters to a list of things to be docsms published.  i.e. it's the first place where we can do a yaml conditional.